en/components/linear_progress.md

```diff
@@ -9,29 +9,267 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
+    </button>
+</div>
+...
+```
+
+And do not forget to firts import `IgxTextAlign` in your component.
+
+```typescript
+import { ..., IgxTextAlign } from "igniteui-angular/main";
+.....
+export class LinearProgressbarSample2Component implements OnInit {
+  @ViewChildren(IgxLinearProgressBarComponent, { read: IgxLinearProgressBarComponent })
+  public linearBars: QueryList<IgxLinearProgressBarComponent>;
+
+  public disable = false;
+  public interval: any;
+  public positionCenter: IgxTextAlign;
+  public positionEnd: IgxTextAlign;
+
+  public ngOnInit() {
+    this.positionCenter = IgxTextAlign.CENTER;
+    this.positionEnd = IgxTextAlign.END;
+  }
+
+  public updateValue() {
+    this.disable = true;
+    this.linearBars.map((bar) => bar.value += this.randomIntFromInterval(1, 3));
+    const shouldStop = this.linearBars.toArray().every((bar) => bar.value >= bar.max);
+    if (shouldStop) {
+    this.disable = false;
+    this.interval = clearInterval(this.interval);
+    }
+  }
+
+  public tick() {
+    if (this.interval) {
+        this.interval = clearInterval(this.interval);
+        this.disable = false;
+        return;
+    }
+    this.interval = setInterval(this.updateValue.bind(this), 60);
+  }
+
+  public reset() {
+    this.linearBars.toArray().forEach((bar) => bar.value = 0);
+  }
+
+  private randomIntFromInterval(min: number, max: number) {
+      return Math.floor(Math.random() * (max - min + 1) + min);
+  }
+}
+```
+
+And now let's see it in the browser:
+
+<div class="sample-container loading" style="height:700px">
+    <iframe id="linear-sample-2-iframe" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/linear-progressbar-sample-2" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="linear-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
 <div class="divider--half"></div>
 
 ### API
 | Name   |       Type      |  Description |
 |:----------|:-------------:|:------|
-| `max` |  number | Set maximum value that can be passed. |
+| `max` |  number | Set maximum value that can be passed. By default it is set to 100. |
 | `type` |  string | Set type of the linear bar. Possible options - `default`, `success`, `info`, `warning`, and `danger`. |
 | `value` |  number | Set value that indicates the completed bar position. |
 | `striped` |  boolean | Set bar to have striped style. |
 | `animate` |  boolean | animation on progress bar. |
+| `textAlign` | enum | Set the position that define where the text is aligned. Possible options - `IgxTextAlign.START` (default), `IgxTextAlign.CENTER`, `IgxTextAlign.END`. |
+| `textVisibility` | boolean | Set the text to be visible. By default is set to `true`. |
+| `textTop` | boolean | Set the position that defene is text to be aligned above the progress line. By default is set to `false`. |
```

Comment:
Set the position that defines if the text should be aligned above the progress line.
Or at least that's what I think you're trying to say?

Reply:
Yes, this property is used to define the text-top position